### ISSUE TITLE
Missing images when added through Tailwind's bg-[] classes when using unstable_dev

### ISSUE THREAD

**Issue by KostarSf** [remix-run/remix#6332]
### What version of Remix are you using?

1.16.0

### Are all your remix dependencies & dev-dependencies using the same version?

- [X] Yes

### Steps to Reproduce

Create simple Remix project with TS
Enable future flag `unstable_dev` in `remix.config.js`
Add Tailwind using [this](http://remix.run/docs/en/main/guides/styling#tailwind-css) instruction
Put any image in the project's `public` folder, say `test.jpg`
Insert this code somewhere in `_index.tsx`

```jsx
<div className="w-24 h-24 bg-slate-300 bg-[url(/test.jpg)] bg-cover"></div>
```

Run dev server

### Expected Behavior

The image should appear on the page

### Actual Behavior

The image does not appear on the page. And if you look at the image's URL it will be `localhost:3001/test.jpg`.
If you remove `unstable_dev` flag and restart dev server the image will appear on the page and its URL will be `localhost:3000/test.jpg`

**Comment by pcattori** [comment]
Should be fixed by #6306 . You can try `@nightly` release if you want to confirm.

**Comment by KostarSf** [comment]
Yes, it works fine with `@nightly`, thank you
